var/spack/repos/builtin/packages/graph500/package.py

```diff
@@ -21,6 +21,8 @@ class Graph500(MakefilePackage):
     def edit(self, spec, prefix):
         makefile = FileFilter(join_path(self.build_directory, 'Makefile'))
         makefile.filter(r'^MPICC\s*=.*', 'MPICC={0}'.format(spec['mpi'].mpicc))
+        if spec.satisfies('%gcc@10.2.0:'):
+            makefile.filter(r'^CFLAGS\s*=\s*', 'CFLAGS = -fcommon ')
```

Comment:
This should probably be implemented as a spack "flag handler" (grepping the packages for `fcommon` shows several use cases). Additionally this is a problem with major version GCC10, not minor version 10.2, so you should extend the applicability.

```
    def flag_handler(self, name, flags):
        wrapper_flags = None

        if name == 'cflags':
            if self.spec.satisfies('%gcc@10:'):
                wrapper_flags = ['-fcommon']

        return (wrapper_flags, None, flags)
```

Reply:
done